ast/Trees.h

```diff
@@ -633,7 +634,7 @@ EXPRESSION(RestArg) {
 
     void _sanityCheck();
 };
-CheckSize(RestArg, 16, 8);
+CheckSize(RestArg, 24, 8);
```

Comment:
I would really love to avoid this—we're already anticipating running into memory problems.

Is it possible to solve this without storing more things in trees?